postgres-driver/sqlc/schema.sql

```diff
@@ -19,29 +20,37 @@ CREATE  TABLE portal_region (
  );
 
 CREATE  TABLE relay (
-	relay_id             bigint  NOT NULL GENERATED ALWAYS AS IDENTITY  ,
-	chain_id             integer  NOT NULL  ,
-	endpoint_id          integer  NOT NULL  ,
+	id             		bigint  NOT NULL GENERATED ALWAYS AS IDENTITY  ,
+	relay_id 			varchar NOT NULL UNIQUE ,
+	pokt_chain_id       varchar  NOT NULL  ,
+	endpoint_id          varchar  NOT NULL  ,
 	session_key   		varchar  NOT NULL  ,
+	relay_source_url 	varchar  NOT NULL  ,
 	pokt_node_address    varchar  NOT NULL  ,
+	pokt_node_domain 	varchar   NOT NULL ,
+	pokt_node_public_key varchar   NOT NULL ,
```

Comment:
`pokt_node_public_key` could be a `CHAR(64)` as all the public keys have that fixed amount of characters. Same for `pokt_node_addrerss` which could be `CHAR(40)` but up to you